climada/entity/measures/measure_set.py

```diff
@@ -444,47 +444,58 @@ def read_att_excel(measures, dfr, var_names):
             """Read Excel measures attributes"""
             num_mes = len(dfr.index)
             for idx in range(0, num_mes):
-                meas = Measure()
+                try:
+                    haz_type = dfr[var_names['col_name']['haz']][idx]
+                except KeyError:
+                    haz_type = ""
 
-                meas.name = dfr[var_names['col_name']['name']][idx]
+                # Search for (a, b) values, put a=1 otherwise
                 try:
-                    meas.haz_type = dfr[var_names['col_name']['haz']][idx]
+                    hazard_inten_imp = (dfr[var_names['col_name']['haz_int_a']][idx],
+                                        dfr[var_names['col_name']['haz_int_b']][idx])
                 except KeyError:
-                    pass
-                meas.color_rgb = np.fromstring(
-                    dfr[var_names['col_name']['color']][idx], dtype=float, sep=' ')
-                meas.cost = dfr[var_names['col_name']['cost']][idx]
-
-                meas.hazard_freq_cutoff = dfr[var_names['col_name']['haz_frq']][idx]
-                meas.hazard_set = dfr[var_names['col_name']['haz_set']][idx]
-                # Search for (a, b) values, put a = 1 otherwise
+                    hazard_inten_imp = (1, dfr['hazard intensity impact'][idx])
+
                 try:
-                    meas.hazard_inten_imp = (dfr[var_names['col_name']['haz_int_a']][idx],
-                                             dfr[var_names['col_name']['haz_int_b']][idx])
+                    exposures_set = dfr[var_names['col_name']['exp_set']][idx]
                 except KeyError:
-                    meas.hazard_inten_imp = (1, dfr['hazard intensity impact'][idx])
+                    exposures_set = ""
```

Comment:
Would it not make more sense to not set it then and leave the default behavior to the `__init__`? Not sure how this would be best written though. No big deal, one can leave it as is too.